lib/SILOptimizer/Mandatory/AddressLowering.cpp

```diff
@@ -2939,11 +2925,18 @@ class DefRewriter : SILInstructionVisitor<DefRewriter> {
     addrMat.initializeComposingUse(&initExistentialValue->getOperandRef());
   }
 
-  // Project an opaque value out of a box-type existential.
   void visitOpenExistentialBoxValueInst(
-      OpenExistentialBoxValueInst *openExistentialBox) {
-    // FIXME: Unimplemented
-    llvm::report_fatal_error("Unimplemented OpenExistentialBoxValue def.");
+      OpenExistentialBoxValueInst *openExistentialBoxValue) {
```

Comment:
Is the operand of open_existential_box_value always a loadable type (a box existential)? If not, then it also needs to be handled in the use rewriter.... just double checking out of paranoia.

Reply:
Looking at the verifier, I think this is the case -  https://github.com/apple/swift/blob/66b08a8a6cce969f1e3e90280b7b40d79cc056eb/lib/SIL/Verifier/SILVerifier.cpp#L3703